src/element.cc

```diff
@@ -9,13 +9,18 @@
 #include "quadrilateral_gimp_element.h"
 #include "quadrilateral_lme_element.h"
 #include "triangle_element.h"
+#include "triangle_lme_element.h"
 
 // Triangle 3-noded element
 static Register<mpm::Element<2>, mpm::TriangleElement<2, 3>> tri3("ED2T3");
 
 // Triangle 6-noded element
 static Register<mpm::Element<2>, mpm::TriangleElement<2, 6>> tri6("ED2T6");
 
+// Triangle LME element
+static Register<mpm::Element<2>, mpm::TriangleLMEElement<2>> quad_lme(
```

Comment:
Need to choose another name. `quad_lme` is already taken. Maybe `tri_lme`>?

Reply:
jajaj sorry